source/shared/cpp/ObjectModel/Enums.cpp

```diff
@@ -128,6 +128,7 @@ namespace AdaptiveSharedNamespace
             {AdaptiveCardSchemaKey::Requires, "requires"},
             {AdaptiveCardSchemaKey::RichTextBlock, "richTextBlock"},
             {AdaptiveCardSchemaKey::Right, "right"},
+            {AdaptiveCardSchemaKey::Schema, "$schema"},
```

Comment:
have you verified that the existing shared model tests pass? also, we should probably have a card-level additional properties test...

Reply:
The shared model tests are passed & new tests for card level additional properties are added to  `AdditionalPropertiesTest.cpp`